packages/devtools_app/lib/src/flutter/screen.dart

```diff
@@ -87,8 +109,11 @@ enum DevToolsScreenType {
   debugger,
   network,
   simple,
+  conditional,
```

Comment:
add a TODO to check if we are using the screen type for logging? we'll probably want to stop doing that and start just using string ids and all conditional screens would get grouped together which wouldn't be great.

Reply:
The original reason this enum was created was so that we could import snapshots, and then based on a string in the snapshot, we could create the correct `Screen` class. The challenge is to make this change while leaving the functionality in [import_export.dart](https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/config_specific/flutter/import_export/import_export.dart#L59) in tact, and also making conditional screens  compatible with import / export.

Reply:
Latest commit includes changes. PTAL